### ISSUE TITLE
GoPlay Episode Fails to Start

### ISSUE THREAD

**Issue by pejeio** [retrospect-addon/plugin.video.retrospect#1848]
### Describe the bug
When attempting to start the episode "1x01. Celebrity Klopjacht - S1 - Aflevering 1", the loading spinner briefly appears and then stops, but the episode does not start afterward.

## Expected Result
The episode should start playing after the loading spinner completes.

## Actual Result
The loading spinner stops, and the episode does not play.

### To Reproduce
1. Go to Retro
2. National
3. GoPlay
4. Vier
5. Celebrity Klopjacht
6. Select any episode.

### Debuglog
The **debug** retrospect.log can be found here:
[retrospect.log](https://github.com/user-attachments/files/17421804/retrospect.log)

### Your Environment
Used Operating system:
OSX

Used versions:
- Kodi version: `21.0.0`
- Retrospect version: `v5.7.18`

**Comment by basrieter** [comment]
Did you update to the latest Widevine version?

**Comment by pejeio** [comment]
> Did you update to the latest Widevine version?

Yes, I received an "Update Widevine" dialog, confirmed the installation, and it completed successfully.

**Comment by basrieter** [comment]
Did you reboot afterwards and then tried again?

**Comment by pejeio** [comment]
> Did you reboot afterwards and then tried again?

Yes, I did.

**Comment by basrieter** [comment]
In that case I need a full DEBUG kodi.log and full DEBUG retrospect.log (the one you posted was at INFO level).

**Comment by pejeio** [comment]
> In that case I need a full DEBUG kodi.log and full DEBUG retrospect.log (the one you posted was at INFO level).

There you go:
[kodi.log](https://github.com/user-attachments/files/17429339/kodi.log)
[retrospect.log](https://github.com/user-attachments/files/17429346/retrospect.log)

**Comment by basrieter** [comment]
Seems like Kodi cannot decrypt the streams. Are you trying to play them from within Belgium or abroad?

**Comment by pejeio** [comment]
> Seems like Kodi cannot decrypt the streams. Are you trying to play them from within Belgium or abroad?

From within Belgium.

**Comment by LeEcHeR123** [comment]
Same here, can't open any episodes on GoPlay anymore.

**Comment by basrieter** [comment]
Can you try https://github.com/retrospect-addon/plugin.video.retrospect/archive/refs/heads/master.zip

**Comment by LeEcHeR123** [comment]
Thanks, Bas! That worked. I see it’s still version 5.7.20. Why is this version working, while the version I had (also 5.7.20) wasn’t?

**Comment by basrieter** [comment]
Because I did not bumped the version yet. Found the problem yesterday and already fixed it in the 'master' branch. Will do an official release soon.

**Comment by AlfaStation1** [comment]
Indeed, Vier has not worked for almost 2 weeks now, both on Kodi for windows and on my 2 android tv-boxes.
The master.zip file posted above fixed the problem.

Thx @basrieter for your wonderfull work!

**Comment by GillesMarkvoort** [comment]
This one can be closed. Works perfectly on the latest release.
